.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,77 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build Job
+
+    runs-on: ubuntu-20.04
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install aiortc native dependencies
```

Comment:
@bhavikapanara we don't need aiortc for this package. That's a webrtc p2p communication library.

Reply:
done